lib/seccomp/CSystemCallFilter_MacOSX.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+
+#include <paths.h>
+#include <cstdio>
+#include <fstream>
+#include <sandbox.h>
+
+namespace ml {
+namespace seccomp {
+
+// The Sandbox rules deny all actions apart from creating fifos,
+// opening files and reading and writing.
+// (allow file-write*) is required for mkfifo and that permission
+// can not be set using the more granular controls.
+static const std::string SANDBOX_RULES("\
```

Comment:
The modern way is to use an unnamed namespace rather than static.

The functions below can also go in it so they get internal linkage.